CONTRIBUTING.md

```diff
@@ -2,7 +2,78 @@
 
 You'll need a copy of `pixi` and `git` in your machine. Then:
 
-1. Clone this repo to disk.
+1. Fork and clone this repo to disk.
 2. `pixi run test` to run the tests. Choose your desired Python version by picking the adequate environment.
 3. `pixi run lint` to run the pre-commit linters and formatters.
 4. `pixi run docs` to build the docs and `pixi run serve` to serve them in your browser.
```

Comment:
I think these should move under "With Pixi` below, maybe with a dedicated subsection "Included tasks".